rules/windows/process_creation/proc_creation_win_adplus_memory_dump.yml

```
@@ -8,10 +8,11 @@ references:
     - https://twitter.com/nas_bench/status/1534915321856917506
 author: Nasreddine Bencherchali (Nextron Systems)
 date: 2022-06-09
-modified: 2023-06-23
+modified: 2025-04-09
 tags:
     - attack.defense-evasion
     - attack.execution
+    - attack.credential_access
```

```suggestion
    - attack.credential-access
```
https://github.com/SigmaHQ/sigma-specification/blob/main/appendix/sigma-tags-appendix.md#namespace-attack